lib/other/IERC20Extended.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity >=0.5.4 <0.7.0;
+
+/**
+ * Extended ERC20 contract interface to include the decimals property.
+ */
+interface IERC20Extended {
```

Comment:
We need a bigger discussion around the ERC20 contracts we use so I logged [this](https://trello.com/c/agxNtzfQ/138-review-erc20-contracts-used). But to answer your question, OZ's `ERC20` is an implementation of their `IERC20` interface (which only declares the mandatory properties, i.e. no `decimals`). I would prefer to use an interface, rather than a contract because of the much flexible integration that provides. 

I'm going to go back to the ERC20 interface we have used elsewhere from /lib/other/ERC20.sol until we agree on a way forward.

Reply:
The only thing to consider in this original ERC20 we have is the decimals property being a uint256 rather than a uint8 as per the standard. Should we add an assert for the decimals value fitting in a uint8 e.g. `require(_decimals < 256, "TPP: token decimals overflow");`?